BUILDING.md

```diff
@@ -41,14 +41,14 @@ platforms in production.
 |--------------|--------------|----------------------------------|----------------------|------------------|
 | GNU/Linux    | Tier 1       | kernel >= 2.6.32, glibc >= 2.12  | x64, arm             |                  |
 | GNU/Linux    | Tier 1       | kernel >= 3.10, glibc >= 2.17    | arm64                |                  |
-| macOS        | Tier 1       | >= 10.10                         | x64                  |                  |
+| macOS / OS X | Tier 1       | >= 10.10                         | x64                  |                  |
```

Comment:
no spaces around the slashes here and elsewhere, please

Reply:
done.